src/demos/action-bar/basic.html

```diff
@@ -81,7 +81,7 @@ <h3>Tooltip on expand toggle</h3>
                   <calcite-action-bar id="action-bar-test">
                     <calcite-action text="Add" icon="plus"></calcite-action>
                   </calcite-action-bar>
-                  <calcite-tooltip id="tooltip">Hello world!</calcite-tooltip>
+                  <calcite-tooltip label="tooltip" reference-element="tooltip" id="tooltip">Hello world!</calcite-tooltip>
```

Comment:
@julio8a I don't think the reference element should be the same as the tooltip. The reference-element should be what triggers the tooltip. Applies to other places.

Reply:
Thanks! I've updated this.